### PR TITLE
Add PP to front end

### PR DESCRIPTION
**Pull request by BinAccel**
Implement #531 

I tested it on my own site so nothing should suddenly go up in flames. I think.

### CODE REVIEW

File: templates/user/users-table.jade [hunk 1]
```diff
@@ -19,6 +19,15 @@ block after_rank
...
                 span.rating(class='{{ user.rating|rating_class }}') #{user.rating}
 
+block before_point_head
+    if sort_links
+       th.pp: a(href=sort_links['performance_points']) {% trans "PP" %}#{sort_order.performance_points}
+    else
+       th.pp {% trans "PP" %}
+
+block before_point
+    td #{user.performance_points|floatformat:2}
```
Comment (quantum5) [inline templates/user/users-table.jade]: You need a CSS class here.

Comment (BinAccel) [inline templates/user/users-table.jade]: Will do at home

Comment (BinAccel) [inline templates/user/users-table.jade]: Excuse the stupid question @quantum5, but why does L38 not need one?

**Comment by quantum5** [comment]
Notice how there is a `.only('display_rank', 'user__username', 'name', 'points', 'rating')` in the query. You must add pp or it will run one query per row.

**Comment by BinAccel** [comment]
Also going to make PP visible by default on the user's profile page

**Pull request merged**